app/controller.py

```diff
@@ -66,8 +67,10 @@ def changeToSaveAs(self):
     self.host.changeFocusTo(self.host.interactiveSaveAs)
 
   def doCommand(self, ch):
+    #Check the commandSet for the input with both its string and integer representation.
     self.savedCh = ch
-    cmd = self.commandSet.get(ch)
+    keyName = app.curses_util.cursesKeyName(ch)
+    cmd = self.commandSet.get(keyName) or self.commandSet.get(ch)
```

Comment:
How about changing this to look for the integer .get(ch) first and then if that fails, do the cursesKeyName(ch) and the .get(keyName). The reason is that it's less work for the cpu.